pytest_fixtures/core/contenthosts.py

```diff
@@ -269,6 +269,23 @@ def oracle_host(request, version):
         yield host
 
 
+@pytest.fixture
+def bootc_host(request):
+    """Fixture to check out boot-c hosts"""
+    with Broker(
+        **host_conf(request),
+        workflow='deploy-bootc',
+        host_class=ContentHost,
+    ) as host:
+        assert (
+            host[0]
```

Comment:
```suggestion
            host
```
PRT fails with
```
20:09:15  >               host[0]
20:09:15                  .execute(f"echo '{constants.DUMMY_BOOTC_FACTS}' > /etc/rhsm/facts/bootc.facts")
20:09:15                  .status
20:09:15                  == 0
20:09:15              )
20:09:15  E           TypeError: 'ContentHost' object is not subscriptable
```
Did you plan to get multiple hosts (like with `_count` parameter) from this broker run?
The docstring says so but the fixture name and code don't.